include/mbedtls/config.h

```diff
@@ -137,12 +137,25 @@
 /**
  * \def MBEDTLS_HAVE_TIME_DATE
  *
- * System has time.h and time(), gmtime() and the clock is correct.
+ * System has time.h, time(), an implementation for mbedtls_platform_gmtime_r()
+ * (see below), and the clock is correct.
  * The time needs to be correct (not necesarily very accurate, but at least
  * the date should be correct). This is used to verify the validity period of
  * X.509 certificates.
  *
  * Comment if your system does not have a correct clock.
+ *
+ * \note mbedtls_platform_gmtime_r() is an abstraction in platform_util.h that
+ * when called behaves similarly to the gmtime() function from the C standard,
+ * but is thread safe. Mbed TLS will try to identify the underlying platform
```

Comment:
Here and elsewhere: “thread safe” → “thread-safe”

Reply:
Addressed in 921b76d.